config/database.yml

```diff
@@ -21,21 +21,18 @@ test:
   database: storage/test.sqlite3
 
 
+deploy_default: &deploy_default
+  adapter: postgis
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 }.to_i + 1 %>
+  timeout: 5000
```

Comment:
wondering if these are overridden since the `ENV["DATABASE_URL"]` exists, and if so, I should keep them around much in the same way I'm keeping the `url` around?

also url looks like: "pya-production-web.cluster-csjcc2coc9nm.us-east-1.rds.amazonaws.com" 
not like `adapter://user:password@host:port/database_name` which seems more classic, is this an issue?

Reply:
when I removed the [adapter,](https://github.com/codeforamerica/pya/pull/29/commits/d4cc5de0f56b9d3d5d0ac5c1880baf00ba5a0416) i got into trouble, it said i need an adapter